gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageReadOptions.java

```diff
@@ -153,6 +158,13 @@ public static Builder builder() {
      */
     public abstract Builder setFastFailOnNotFound(boolean fastFailOnNotFound);
 
+    /**
+     * If false then reading a file with GZIP content encoding (HTTP header "Content-Encoding:
+     * gzip") will result in failure (IOException is thrown). If true then GZIP-encoded files will
+     * be read successfully.
+     */
+    public abstract Builder setSupportContentEncoding(boolean supportContentEncoding);
```

Comment:
Again, may want to make this reflect gzip.

Reply:
Done.